basics/Pallindrome and digits examples/Check if a nuber is pallindrome.py

```diff
@@ -0,0 +1,17 @@
+import time
+x = input("Type the number to check if it is a pallindrome:")
+
+n = 1
+rev = 0
+
+for i in x:
+    i = int(i)
+    z = i * n
+    n = n * 10
+    rev = rev + z

```

Comment:
This is too complicated for a tutorial about Python basics. Please rewrite this to use `[::-1]` or some other easier way to reverse a string instead of trying to reverse as `int`.

Also, there are 5 programs about reversing and palindromes. We only need one.

Reply:
Ok..
I just wanted to show other methods to check a pallindrome